source/include/core_json.h

```diff
@@ -106,7 +109,6 @@ JSONStatus_t JSON_Validate( const char * buf,
  * @param[in] max  size of the buffer.
  * @param[in] queryKey  The key to search for.
  * @param[in] queryKeyLength  Length of the key.
- * @param[in] separator  A character between a key and a sub-key in queryKey.
```

Comment:
Why was this removed?

Reply:
The array support introduces square brackets to surround an index in a query.  Rather than add code to permit the brackets to be user specified and code to ensure no collisions among separators, I decided that a user-specified separator was a misfeature and removed it.